src/Utilities/cachingoptimizer.jl

```diff
@@ -22,9 +22,9 @@ A `CachingOptimizer` has two modes of operation (`CachingOptimizerMode`):
 - `Manual`: The only methods that change the state of the `CachingOptimizer` are [`resetoptimizer!`](@ref), [`dropoptimizer!`](@ref), and [`attachoptimizer!`](@ref). Attempting to perform an operation in the incorrect state results in an error.
 - `Automatic`: The `CachingOptimizer` changes its state when necessary. For example, `optimize!` will automatically call `attachoptimizer!` (an optimizer must have been previously set). Attempting to add a constraint or perform a modification not supported by the optimizer results in a drop to `EmptyOptimizer` mode.
 """
-mutable struct CachingOptimizer{MT<:MOI.ModelLike} <: MOI.AbstractOptimizer
-    model_cache::MT
-    optimizer::Union{Nothing,MOI.AbstractOptimizer}
+mutable struct CachingOptimizer{OptimizerType, ModelType<:MOI.ModelLike} <: MOI.AbstractOptimizer
+    optimizer::OptimizerType
```

Comment:
Do you plan to update this to `optimizer::Union{OptimizerType, Nothing}`?

Reply:
Ups, fixed